tests/mocked_plans/test_base_plan.py

```diff
@@ -15,20 +15,23 @@
 from unittest.mock import patch
 
 import pytest
+from utils import sample_plans
 
 from cou.commands import CLIargs
 from cou.steps.analyze import Analysis
 from cou.steps.plan import generate_plan
 
 
 @pytest.mark.asyncio
+@pytest.mark.parametrize(
```

Comment:
I think it would be a little more idiomatic to parametrize the sample_plan fixture instead. See https://docs.pytest.org/en/6.2.x/fixture.html#parametrizing-fixtures

Reply:
That would not work prperly, since what I want to create plan and use it in diff type of test e.g. with `@patch("cou.utils.nova_compute.get_instance_count", side_effect=[0, 1, 0])`

Reply:
Now I see, that we are trying to get all files, which again would not be correct. If I create file with plan, which represent plan with some issues on model. I think it would be better to manually write those names here.

Reply:
We still generate the plan for applications that can generate the plan and skip those that fail. I don't see a need to change the test structure. Even if we do need to change something, maybe we can have a separate folder with the problematic models